tests/playwright/src/bootc-extension.spec.ts

```diff
@@ -41,13 +41,14 @@ const imageName = 'quay.io/centos-bootc/fedora-bootc';
 const containerFilePath = path.resolve(__dirname, '..', 'resources', 'bootable-containerfile');
 const contextDirectory = path.resolve(__dirname, '..', 'resources');
 const isLinux = os.platform() === 'linux';
+const cicd = process.env.SKIP_INSTALLATION;
```

Comment:
cicd is not really helpful based on the env var

please rename it to `skipInstallation`

Reply:
done